conf.py

```diff
@@ -38,32 +38,31 @@
     "display_github": True,
     "github_user": "ros-planning",
     "github_repo": "moveit_tutorials",
-    "github_version": "melodic-devel",
+    "github_version": "master",
     "conf_py_path": "",
     "source_suffix": source_suffix,
     "css_files": ['_static/override.css'],
     "favicon": "favicon.ico"
 #  "logo": "logo.png"
 }
 
-# Experimental
-
-
-
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
+ros_distro = 'melodic'
```

Comment:
this should be cherry-picked to master and this string updated to master, also.

Reply:
Agreed. Now that I think about it, I think this should target master instead of melodic. I can cherry pick into melodic